src/main/java/uk/ac/ebi/ega/egafuse/config/EgaFuseApplicationConfig.java

```diff
@@ -53,6 +53,13 @@
 
     @Value("${api.chunksize}")
     private long chunkSize;
+    
+    public static boolean isTreeStructureEnable;
+    
+    @Value("${tree}")
+    public void setFileDatabaseService(boolean isTreeStructure) {
+        isTreeStructureEnable = isTreeStructure;
+    }

```

Comment:
Not sure if this can be skipped and @Value("${api. tree}") can be directly injected to CliConfigurationValues class?

Reply:
It is used in EgaDirectory class and we can inject values directly in it, however, there are a lot of places from where  EgaDirectory is created so it is difficult to inject this in all those classes. So I decided to make it to use as a constant